se/Mortgage/v0/schema/definitions/OwnedProperty.yaml

```diff
@@ -0,0 +1,63 @@
+"$id": "https://open-broker.org/se/Mortgage/v0/schema/definitions/OwnedProperty"
+"$schema": "http://json-schema.org/draft-07/schema#"
+
+title: OwnedProperty
+description: |
+  An owned real-estate property, such as an apartment, house or
+  cottage that the applicant(s) own
+type: object
+additionalProperties: false
+required:
+  - propertyType
+  - assessedValue
+  - sqaureMeters
+  - existingMortgage
+  - monthlyCost
+  - partOfOwnership
+properties:
+  propertyType:
+    type: string
+    title: The property type
+    enum:
+      - HOUSE
+      - TERRACED_HOUSE
+      - VACATION_HOME
+      - APARTMENT
+      - OTHER
+    "meta:enum":
+      HOUSE: An ordinary house with a plot
+      TERRACED_HOUSE: A house that is built adjacent to other houses
+      VACATION_HOME: |
+        A house built for recreational living that may or may not
+        be fit as a permanent resident
+      APARTMENT: An apartment that belongs to condominium compound
+      OTHER: |
+        Some other form of real-estate property that does not belong
+        to any of the above categories
+  assessedValue:
+    type: integer
+    min: 0
+    description: The estimated value of the property, in SEK
+  sqaureMeters:
+    type: integer
+    min: 0
+    description: The number of square meters of the property (excluding plot)
+  existingMortgage:
+    type: integer
+    min: 0
+    description: How mortgaged the property is, in SEK
+  monthlyCost:
+    type: integer
+    min: 0
+    description: |
+      The sum of all montlhy costs associated with the property, such as
+      upkeep or a monthly fee to a condominium compound if the property
+      is an apartment
+  partOfOwnership:
```

Comment:
Do we need consistenci in how we use parts of a whole? Above, in e.g. effective interest rate, we use 1 to represent the whole, and here we use 100.

Reply:
Honestly, that's a good motivation for doing it like it is, and I would suggest to keep it as is. To use Integer 1-100 when exact percentage isn't necessary (like you describe here how ownership isn't).

Reply:
How would 33% be interpreted? Given that 33 + 33 + 33 = 99

Reply:
That it could be around but not (necessarily) exactly 33%. It seems that the exact value is not requested. If I find any API that requires higher precision I will reconsider this, but at the moment an integer seems to be enough.